lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -188,14 +207,28 @@ class ElementScreenshotRenderer {
   /**
    * Renders an element with surrounding context from the full page screenshot.
    * Used to render both the thumbnail preview in details tables and the full-page screenshot in the lightbox.
+   * Returns null if element rect is outside screenshot bounds.
    * @param {DOM} dom
    * @param {ParentNode} templateContext
    * @param {LH.Audit.Details.FullPageScreenshot} fullPageScreenshot
    * @param {LH.Artifacts.Rect} elementRectSC Region of screenshot to highlight.
    * @param {Size} maxRenderSizeDC e.g. maxThumbnailSize or maxLightboxSize.
-   * @return {Element}
+   * @return {Element|null}
    */
   static render(dom, templateContext, fullPageScreenshot, elementRectSC, maxRenderSizeDC) {
+    const fullPageScreenshotRect = {
+      left: 0,
+      top: 0,
+      right: fullPageScreenshot.width,
+      bottom: fullPageScreenshot.height,
+      width: fullPageScreenshot.width,
+      height: fullPageScreenshot.height,
+    };
+    if (!rectContains(fullPageScreenshotRect, elementRectSC)) {
```

Comment:
replace `rectContains(rect1, rect2)` with

```js
function screenshotContainsRect(fullPageScreenshot, rect) {
  return rect.top >= 0 &&
    rect.right <= fullPageScreenshot.width &&
    rect.bottom <= fullPageScreenshot.height &&
    rect2.left >= 0;
}
```

and avoid having to setup this `fullPageScreenshotRect` object?